### PR TITLE
ci: updating supported Python versions

### PR DESCRIPTION
**Pull request by clatapie**
Unsupporting Python 3.8 and supporting Python 3.12.

### CODE REVIEW

File: .github/workflows/ci_cd.yml [hunk 1]
```diff
@@ -75,7 +75,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        python-version: ['3.8', '3.9', '3.10', '3.11']
+        python-version: ['3.9', '3.10', '3.11', '3.12']
```
Comment (RobPasMue) [inline .github/workflows/ci_cd.yml:78]: And 3.9 should be dropped as well, right?

**Review by RobPasMue — CHANGES_REQUESTED** [review]

File: .github/workflows/ci_cd.yml (hunk 1 above)
Comment (clatapie) [reply .github/workflows/ci_cd.yml]: Completely right! Let me fix it

File: pyproject.toml [hunk 2]
```diff
@@ -30,7 +29,7 @@ dependencies = [
     "ansys-tools-path>=0.2.4",
     "pyansys-tools-versioning>=0.3.3",
     "numpy>=1.14.0",
-    "importlib-metadata>=4.0,<5; python_version<='3.8'",
+    "importlib-metadata>=4.0",
```
Comment (RobPasMue) [inline pyproject.toml:32]:
```suggestion
```
This is no longer needed

**Review by RobPasMue — APPROVED** [review]
Small comment - otherwise LGTM

**Comment by RobPasMue** [comment]
Once you merge this, I recommend you do a new minor release 0.2

**Pull request merged**